calcom/cal.com
# fix: Install Stripe for team on team event type app page

## What does this PR do?



This PR installs Stripe for a team if installed from that team's event type.

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?


	
- Create a team event type
- Go to the apps page and install Stripe
- After the OAuth flow, Stripe should be installed for that team

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
